### DISCUSSION TITLE
Disable upload File

### DISCUSSION THREAD

**Discussion by eMerzh** [excalidraw/excalidraw#4825] [Q&A]
Hello, 

not sure if it really fit here or not... but i'm trying to embed excalidraw but i don't really want users to upload binary files
i would rather stay with the json thing ... (and base64 file is not really wanted too).

Is there a way to achieve that? 
maybe hooking in the "on add" or smth?

**Comment by dwelle** [comment]
Do you mean the image upload?

**Reply by eMerzh** [reply]
yes image upload via drag & drop or image button ... sorry of if was not clear

**Reply by dwelle** [reply]
Ok, that's a valid use case that we totally for forgot about. I've created an issue for this: https://github.com/excalidraw/excalidraw/issues/4831
